frontend/awx/views/jobs/hooks/useJobsColumns.tsx

```diff
@@ -62,14 +63,14 @@ export function useJobsColumns(options?: { disableSort?: boolean; disableLinks?:
       },
       {
         header: t('Started'),
-        cell: (job: UnifiedJob) => job.started && <SinceCell value={job.started} />,
+        cell: (job: UnifiedJob) => job.started && <p>{formatDateString(job.started)}</p>,
```

Comment:
Could we enhance the SinceCell?
- Rename to DateTimeCell
- Add a format prop: "since" | "date-time"

That way any other places that need the same formatting can use that component.